syntax = "proto3";

package replication;

service Sequence {
  rpc Write (WriteRequest) returns (WriteResponse);
}

message WriteRequest {
  string key = 1;
  string value = 2;
}

message WriteResponse {
  string ack = 1;
} 


